java/fury-core/src/main/java/org/apache/fury/resolver/EnumStringResolver.java

```diff
@@ -0,0 +1,46 @@
+package org.apache.fury.resolver;
+
+import org.apache.fury.memory.MemoryBuffer;
+import org.apache.fury.meta.MetaString;
+
+import static org.apache.fury.meta.Encoders.GENERIC_DECODER;
+import static org.apache.fury.meta.Encoders.GENERIC_ENCODER;
+
+public class EnumStringResolver {
```

Comment:
Why not use MetaStringResolver directly

Reply:
because org.apache.fury.resolver.MetaStringResolver#getOrCreateMetaStringBytes is not public and MetaStringBytes annotated by @Internal. I feel like I shouldn't break the existing design

Reply:
You can use it in enum serialization, `@internal` just means users' serializer should not use this API

Reply:
Ok, so i can add public before org.apache.fury.resolver.MetaStringResolver#getOrCreateMetaStringBytes？

Reply:
And I also need to add public before MetaStringBytes class, are you sure I can do this? 😂